fighters/falco/src/acmd/specials.rs

```diff
@@ -93,10 +97,16 @@ unsafe fn falco_special_air_s_end_game(fighter: &mut L2CAgentBase) {
  unsafe fn falco_special_hi_hold(fighter: &mut L2CAgentBase) {
         let lua_state = fighter.lua_state_agent;
         let boma = fighter.boma();
-    frame(lua_state, 20.0);
+        frame(lua_state, 1.0);
+        if is_excute(fighter) {
+            KineticModule::unable_energy_all(boma);
+        }
```

Comment:
This will make him unable to grab ledge during upB startup, when we implement it